OBASubmitReportViewController.m

```diff
@@ -19,19 +23,54 @@ @implementation OBASubmitReportViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+  self.submitButton.backgroundColor = OBADARKGREEN;
+  self.navigationController.title = @"Submit Report";
   
-  self.busInfoLabel.text = @"Hello, user!"; //replace with bus information from previous screen
-  
+  [self.segmentedControl setTitleTextAttributes:@{NSFontAttributeName: [UIFont fontWithName:@"Arial" size:16.0],
+                                                 NSForegroundColorAttributeName: [UIColor blackColor]}
+                                       forState: UIControlStateNormal];
 }
 
 - (void)didReceiveMemoryWarning {
     [super didReceiveMemoryWarning];
 }
 
-- (IBAction)busFullSwitch:(id)sender {
+- (IBAction)submitButtonPress:(id)sender {
+  OBAProblemReport *problemReport = [OBAProblemReport object];
+  problemReport.tripID = _selectedArrivalAndDeparture.tripId;
+  problemReport.problemReportType = OBAProblemReportTypeFullBus;
+  
+  NSString *commentText = [[NSString alloc] init];
+  commentText = _commentTextField.text;
+  problemReport.comments = commentText;
```

Comment:
`problemReport.comments = _commentTextField.text;`, and delete the two previous lines